CalibTracker/SiStripHitResolution/plugins/SiStripHitResolFromCalibTree.cc

```diff
@@ -0,0 +1,1752 @@
+//Original Author:  Christopher Edelmaier
+//        Created:  Feb. 11, 2010
+#include <memory>
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+#include "CalibFormats/SiStripObjects/interface/SiStripDetCabling.h"
+#include "CalibFormats/SiStripObjects/interface/SiStripQuality.h"
+#include "CalibTracker/Records/interface/SiStripDetCablingRcd.h"
+#include "CalibTracker/Records/interface/SiStripQualityRcd.h"
+#include "CalibTracker/SiStripCommon/interface/SiStripDetInfoFileReader.h"
+//#include "CalibTracker/SiStripHitEfficiency/interface/TrajectoryAtInvalidHit.h"
```

Comment:
Can this commented out line get removed?

Reply:
done in https://github.com/cms-sw/cmssw/pull/40754/commits/01bbe6e597150b216293e2dc33cdada65a192d3d.